pkgs/development/python-modules/treq/default.nix

```diff
@@ -33,13 +33,12 @@ buildPythonPackage rec {
     trial treq
   '';
 
-  doCheck = false;
-  # Failure: twisted.web._newclient.RequestTransmissionFailed: [<twisted.python.failure.Failure OpenSSL.SSL.Error: [('SSL routines', 'ssl3_get_server_certificate', 'certificate verify failed')]>]
+  patchPhase = ''
```

Comment:
Please always use `postPatch` when adding additional patching. That way, one can still pass in `patches`.